libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -245,7 +245,24 @@ SymbolicTupleVariable::SymbolicTupleVariable(
 	solAssert(m_sort->kind == Kind::Tuple, "");
 }
 
-vector<SortPointer> const& SymbolicTupleVariable::components()
+smtutil::Expression SymbolicTupleVariable::currentValue(frontend::TypePointer const& _targetType) const
+{
+	if (!_targetType || sort() == smtSort(*_targetType))
+		return SymbolicVariable::currentValue();
+
+	auto const& thisTuple = dynamic_cast<TupleSort const&>(*sort());
+	auto const& otherTuple = dynamic_cast<TupleSort const&>(*smtSort(*_targetType));
+	solAssert(thisTuple.components.size() == otherTuple.components.size(), "");
```

Comment:
soltest seems to throw here.

Reply:
Yes, but only on Mac OS. No idea why. @chriseth mentioned it's not a new thing

Reply:
Yeah, I was probably wrong.

Reply:
I tried debugging this on CI Mac OS, but:
- I can't use `gdb` there because it requires some signing (???). I did the signing following google stuff but the signing just hangs.
- When I use `lldb` the run hangs. Running `solc` works fine, I get the exception, but when running anything inside `lldb`, even without the SMTChecker, it hangs.
So I open to suggestions...

Reply:
If debuggers don't work, one way around the problem is to add a bunch of `cout` and run it on the CI? Not sure if `soltest` will suppress it though.

Reply:
Yea, I'll try that. Hard to know the source of the problem without a stack trace though